theme/themes/pastanaga/elements/button.overrides

```diff
@@ -26,6 +26,14 @@
   }
 }
 
+#main {
+  .ui.basic.buttons .button,
```

Comment:
@iFlameing A last thing, for what is this? It seems to me too much general, we need to add specification.

Reply:
I really don't know. I don't make any modification in button.overrides.

Reply:
Ok, done!